x/evm/statedb/statedb.go

```diff
@@ -85,6 +85,11 @@ func (s *StateDB) Keeper() Keeper {
 	return s.keeper
 }
 
+// GetContext returns the transaction Context.
+func (s *StateDB) GetContext() sdk.Context {
```

Comment:
I'm curious as to why it was added.

Reply:
https://github.com/xpladev/ethermint/pull/14/commits/0dd3e83b3d83abc20727e63bc0dcb5f5c3703dde